README.md

```diff
@@ -139,18 +139,22 @@ The table below describes all the available configuration values for Waggle Danc
 | Property                                              | Required | Description |
 |:----|:----:|:----|
 | `primary-meta-store`                                  | No       | Primary MetaStore config. Can be empty but it is advised to configure it. |
-| `primary-meta-store.remote-meta-store-uris`           | Yes      | Thrift URIs of the federated read-only metastore. |
```

Comment:
Not all metastore have a remote-meta-store-uri (e.g. glue), this is a hive thing and shouldn't be in our AbstractMetaStore definition, it should be optional